src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.model.Figure;
+import core.basesyntax.services.ColorSupplier;
+import core.basesyntax.services.FigureSupplier;
+
+public class Circle extends Figure {
+    // Площа = π × r2
```

Comment:
Thanks for the comments, but the code should be understandable without them
```suggestion
```

Reply:
Sorry, those were not for you, but for me. I just forgot to delete them.